plugins/org.jboss.tools.jst.web.kb/src/org/jboss/tools/jst/web/kb/internal/validation/ELValidator.java

```diff
@@ -510,7 +510,7 @@ private void validateElOperand(ELReference elReference, ELInvocationExpression o
 		if(token==null) {
 			WebKbPlugin.getDefault().logError("The token from unresolved segment is null. EL: [" + operand + "]");
 		}
-		varName = token.getText();
+		varName = token != null ? token.getText() : null;

```

Comment:
It's matter of your personal preferences. The logic will be the same. So feel free to use "return".